.github/workflows/main.yml

```diff
@@ -15,18 +15,17 @@ jobs:
 
       matrix:
         ruby-version:
-          - 3.2
-          - 3.1
-          - 3.0
-          - 2.7
-          - 2.6
-          - 2.5
-          - 2.4
-          - 2.3
-          - jruby-9.4.3.0
-          - jruby-9.2.14.0
-          - truffleruby-23.0.0
-          - truffleruby-22.1.0
+          - "3"
+          - "3.2"
+          - "3.1"
```

Comment:
@PetrHeinz can you also add `3.3` and `3.4` since you're at it?

Reply:
```suggestion
          - "3.4"
          - "3.3"
          - "3.2"
          - "3.1"
```